tests/FakeItEasy.Specs/ConfigurationSpecs.cs

```diff
@@ -184,20 +184,255 @@ public interface IFoo
                 .x(() => exception.Should().BeAnExceptionOfType<InvalidOperationException>());
         }
 
+        [Scenario]
+        public static void CallToObjectOnNonFake(
+            BaseClass notAFake,
+            Exception exception)
+        {
+            "Given an object that is not a fake"
+                .x(() => notAFake = new BaseClass());
+
+            "When I start to configure the object"
+                .x(() => exception = Record.Exception(() => A.CallTo(notAFake)));
+
+            "Then it throws an argument exception"
+               .x(() => exception.Should().BeAnExceptionOfType<ArgumentException>()
+                   .And.Message.Should().Contain("The specified object is not recognized as a fake object."));

```

Comment:
Out of scope for this PR, but I think error messages like this one should be extracted to resources. Right now they're repeated in many places (especially in tests/specs), so if we want to change them, we'll have to do it in multiple places.